src/style.css

```diff
@@ -0,0 +1,69 @@
+* {
```

Comment:
As mentioned in some other review don't use `*` for resetting styles. You don't have control what tags are reset, this may lead to some undesired behaviour. Consider creating separate css file for reset purposes and importing it here. There in the reset.css, list all tags and reset their desired styles. Ideally you would use something like [Meyer Reset](https://meyerweb.com/eric/tools/css/reset/)